packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -4059,6 +4059,7 @@ The following Factor types are supported:
 | `u2f`                 | Hardware Universal 2nd Factor (U2F) device                                           |
 | `web`                 | HTML inline frame (iframe) for embedding verification from a third party                                              |
 | `webauthn`            | Hardware WebAuthn device                                                  |
```

Comment:
I am intentionally being a bit verbose here since we do field a lot of questions (both internally and externally) about the conditions under which `signed_nonce` shows up.